MANIFEST.in

```diff
@@ -1,10 +1,4 @@
 include README.md
 include LICENSE
-include requirements.txt
-include pybert/images/icon.png
-recursive-include docs/
-recursive-include tests/
-global-exclude *.swp
-global-exclude *.DS_Store
-global-exclude *.buildinfo
```

Comment:
Why did you remove these 3 global excludes?

Reply:
I think that was a copy paste error on my part; will add back!

Reply:
fixed